ja2003.appdata.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.christianrauch.Jahresarbeit-2003</id>
+  
+  <name>Jahresarbeit 2003</name>
+  <summary>Find die größte Fläche von zusammenhängenden Quadraten</summary>
+
+  <url type="homepage">https://github.com/christianrauch/Jahresarbeit-2003</url>
+  
+  <project_license>GPL-2.0</project_license>
+  <project_license>OFL-1.1</project_license>
+  <project_license>Apache-2.0</project_license>
```

Comment:
there should only be one of these tags.

Reply:
Ok, in this case I combined them with `AND` as suggested at https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-project_license.